tests/run/common_utility_types.srctree

```diff
@@ -36,3 +36,12 @@ assert type(a.funcA) is type(b.funcB)
 
 assert a.funcA.func_globals is a.__dict__
 assert b.funcB.func_globals is b.__dict__
+
+# Test that it's possible to look up the name of the class
+from sys import modules
+for n, mod in modules.items():
+    if n.startswith("_cython_"):  # the internal module..
```

Comment:
Note that there might be more than one such module, e.g. when `numpy.random` is imported.

Reply:
True - however, this is run in a srctree test that starts a separate process and imports almost nothing so I think it's safe. I've added an assert to confirm that.

I did try looking up `type(func).__module__` but that fails with type-specs because `__module__` is an accessor rather than a string. So I've reverted to a assuming there will only be one `_cython_` module.